src/utils.py

```diff
@@ -175,19 +177,18 @@ def collect_user_details(request_header):
     # Get refresh frequency
     refresh_freq = input('How often do you want to refresh the calendar (in seconds)? Default 15. Minimum 5. : ')
     refresh_freq = int(refresh_freq) if refresh_freq and int(refresh_freq) >= 5 else 15
-
+    
     # Get search start date
     start_date = input(
-        '\nSearch for next seven day starting from when?\nUse 1 for today, 2 for tomorrow, or provide a date in the format DD-MM-YYYY. Default 2: ')
+        '\nSearch for next seven day starting from when?\nUse 1 for today, 2 for tomorrow, or provide a date in the format yyyy-mm-dd. Default 2: ')
```

Comment:
I think we should consider reverting this change. The default time which everyone is familiar with in India is `DD-MM-YYYY`. Changing this to `yyyy-mm-dd` might not be super intuitive for users.